src/screens/FeedScreen/RenderList.js

```diff
@@ -131,10 +131,7 @@ const RenderListFeed = (props) => {
 
   const topicBottomPosition = () => {
     if (hasComment) {
-      if (getTotalReaction(item) === 1) {
-        return isBlurred && getHeightReaction() + normalize(4);
-      }
-      return getHeightReaction();
+      return getHeightReaction() + normalize(4);
     }
     return getHeightFooter();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new implementation of `topicBottomPosition` function seems to be simplified, but it might not cover all the cases that were handled in the old implementation. The old code checks if the total reaction is 1 and if the item is blurred before adding the normalized value. If these conditions are still relevant, they should be included in the new implementation.
```diff
-    if (hasComment) {
-      return getHeightReaction() + normalize(4);
-    }
+    if (hasComment) {
+      if (getTotalReaction(item) === 1 && isBlurred) {
+        return getHeightReaction() + normalize(4);
+      }
+      return getHeightReaction();
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->